### PR TITLE
 Chore: Added more Git CheatSheet for Developers

### PR DESCRIPTION
**Pull request by Eazyisreal**
### Description
Added more Git CheatSheet for Developers

### Checklist:
- [x] Read our [contributing guidelines](../docs/CONTRIBUTING.md).
- [ ] Search for duplicates.
- [ ] Used an informative name for this pull request.

### Follow-up
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: CheatSheets/git-cheatsheet.md [hunk 1]
```diff
@@ -8,7 +8,9 @@ Table of Contents
   - [Branching & Merging](#branching--merging)
   - [Sharing & Updating Projects](#sharing--updating-projects)
   - [Inspection & Comparison](#inspection--comparison)
+  - [Tracking Path Changes](#tracking--path--changes)
   - [Setting up Alias](#setting-up-alias)
+  - [Rewrite History](#rewrite--history)
```
Comment (crescentpartha) [inline CheatSheets/git-cheatsheet.md:13]: Links are not working.

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Request for changing.

**Comment by Eazyisreal** [comment]
@crescentpartha I have made all necessary changes

**Review by crescentpartha — APPROVED** [review]
Approved changes

**Pull request merged**